virtcontainers/device/manager/utils.go

```diff
@@ -105,6 +105,11 @@ func isLargeBarSpace(resourcePath string) (bool, error) {
 	suffix := []string{"", "K", "M", "G", "T"}
 	for rIdx, line := range strings.Split(string(buf), "\n") {
 		cols := strings.Fields(line)
+		// start and end columns are required to calculate the size
```

Comment:
Looks good. 
Just wondering if its documented somewhere what this format should like @Jimmy-Xu ?